src/storageClients/BlobStorageManager.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+const Promise = require('bluebird');
+const azure = Promise.promisifyAll(require('azure-storage'));
+
+const BLOB_STORAGE_CONNECTION_STRING = process.env.BLOB_STORAGE_CONNECTION_STRING;
+
+module.exports = {
+
+  Get: function(containerName, blobName, id) {
+    let blobSvc = azure.createBlobService(BLOB_STORAGE_CONNECTION_STRING);
+    return new Promise((resolve, reject) => {
+      blobSvc.getBlobToTextAsync(containerName, blobName, null)
+        .then(text => {
+          let item = JSON.parse(text)[id];
```

Comment:
Oh true, so I should break this into two steps. I think I forgot to do this when promisify-ing.